server/db/models/orders.js

```diff
@@ -0,0 +1,25 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Orders = db.define('orders', {
+  quantity: {
+    type:Sequelize.INTEGER
+  },
+  price: {
+    type:Sequelize.INTEGER
+  },
+  timestamp: {
+    type:Sequelize.DATE
+  },
+  status: {
+    type: Sequelize.ENUM('Created', 'Processing', 'Cancelled', 'Completed')
+  },
+  billingAddress: {
+    type: Sequelize.TEXT,
+  },
+  shippingAddress: {
+    type: Sequelize.TEXT
+  }
+})
+
```

Comment:
add payment method (add sequelize validation to check if valid credit card number)